styles/trinket/catppuccin.user.css

```diff
@@ -0,0 +1,1280 @@
+      );
+
+      width: 0 !important;
+      height: 0 !important;
+      background-image: url("data:image/svg+xml;utf8,@{svg}") !important;
+      background-repeat: no-repeat !important;
+    }
+
+    img[src$="trinket-logo.png"] {
```

Comment:
Are these selectors meant to be using `$=`?

Reply:
yeah, it's matching just the filename instead of the whole url

Reply:
the images themselves are at `/cache-prefix-random_hex/img/...` so using just `=` wouldn't work

Reply:
I understand, just wanted to make sure you did too :)